charts/datastore/values.yaml

```diff
@@ -0,0 +1,39 @@
+# Default values for datastore.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: rookout/data-on-prem
+  pullPolicy: IfNotPresent
+
+datastore:
+  env: production
+  serverMode: AUTOTLS
+  publicKey: LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUFwMnFkMkJER1Z1Z2ludUtPT29wMwpJb0ZPZ1BBSUFzTXJrUnlxajVkWnZDT2FBcjR6VDBkeW81YUM4UUxQUVRha1JGYUdOR1dRcTc4RmlraWNlRVJvCkluMnZQUW9scXpiT1kwcVhLWHdMSlZ5TmwzZFhGVXJCSVdONnIveGk2UWo4OGJqOElpd0tnTlIvZ3BSaG82ZG0KbldpWUpja202bjYwV3dGcmpodGpXVllOWE9VWWRycEt2YVdsc2hHenVpbThOaUJBcXp1Q2EwOEorSWVzWHZRdwpSclZHQzFRTEpHZXJXeGRnVEUwS3duaUFLRVFCeFpJVXZVRXd0RmV1anZSdmlNRnd0WmdKV3BNQll4Mk9xR2Y1CklpVDJ1RXZFUmpXRWtXcG4zVkI1Wkg4QWhpR3F3TkxFNzExTzk4S2xRK2FSVlBUOEJZZHNWM2ZTZGVmZzZXNWUKRFFJREFRQUIKLS0tLS1FTkQgUFVCTElDIEtFWS0tLS0tCg==
+  autoTlsDomain: ''
+  
+service:
+  type: ClusterIp
+  port: 80
+  loadBalancerIP: ''
+
+ingress:
+  enabled: True
+  path: '/*'
+  ingressClass: ''
+
+resources:
+  requests:
```

Comment:
I know that your saying is `"Not everything that is present is inherently the right way"`, but this is the same requirements as the controller on-prem and I don't think we faced any problems regarding the `required` resources.

I agree that it should be more (the dop is minified facade+mgmt so those are my initial inspirations), but seeing the on-prem low requirements makes me question it.

Reply:
Nobody is using it... plus requests only affects scheduling, not actual resource consumption.
32/1000 of 1 CPU is definitely too little. Why do you think this represents the amount of CPU an active DOP will use?

Reply:
I see that in idle times, the mgmt and facade are using less than 100m, so I get from that that more than 100m for `requests.cpu` is unnecessary.

Reply:
The question should be not "how much does it use in idle?" but how much does it use on average -- how much it will need to operate properly given that other processes are also running on the same node. If everything is underprovisioned, then everything running on the node will be slow since it doesn't have enough resources. Since this runs on a customer's node, the requests IMHO should mean how much we expect it to consume when it's under use. Obviously when it's doing nothing, it's doing nothing -- but how is that relevant to our resource usage?